certs_governance/src/storage_types.rs

```diff
@@ -22,15 +22,15 @@ pub enum Status {
 #[contracttype]
 #[derive(Clone, Debug, PartialEq)]
 pub struct CertData {
-    pub id_cert: Bytes,
+    pub cert_id: Bytes,
```

Comment:
Same as organization, maybe it makes more sense to name that field just **id**, since it's inside the cert data, it is understood where the **id** belongs to, ¿what do you think @Jenietoc @einerzg? 🤔